### PR TITLE
Add Stone Kingdoms

### PR DESCRIPTION
**Pull request by melix99**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:** https://gitlab.com/stone-kingdoms/stone-kingdoms
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge). **(The segment after "com.gitlab" is "stonekingdoms" instead of "stone-kingdoms" because the app-id in the manifest doesn't accept dashes other then the last segment)**
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

File: com.gitlab.stonekingdoms.StoneKingdoms.json [hunk 1]
```diff
@@ -0,0 +1,95 @@
+{
+    "app-id": "com.gitlab.stonekingdoms.StoneKingdoms",
```
Comment (hfiguiere) [inline com.gitlab.stonekingdoms.StoneKingdoms.json:2]: it should be `io.gitlab`.

https://docs.flatpak.org/en/latest/conventions.html?highlight=app-id#application-ids

**Review by hfiguiere — COMMENTED** [review]
have the icon, desktop and appstream files been submitted upstream for inclusion in a future release?

File: com.gitlab.stonekingdoms.StoneKingdoms.json (hunk 1 above)
Comment (hfiguiere) [reply com.gitlab.stonekingdoms.StoneKingdoms.json:2]: also it should be `stone_kingdoms`. `-` are replaced with `_`. Same link.

**Comment by melix99** [comment]
> have the icon, desktop and appstream files been submitted upstream for inclusion in a future release?

Oops, not yet, I forgot to open the MR.

**Comment by hfiguiere** [comment]
bot, build io.gitlab.stone_kingdoms.StoneKingdoms

**Comment by melix99** [comment]
> have the icon, desktop and appstream files been submitted upstream for inclusion in a future release?

https://gitlab.com/stone-kingdoms/stone-kingdoms/-/commit/8065caf1ef42e695391c515575dd254b0a7ff408

**Comment by melix99** [comment]
I tested the build and it works well.

**Pull request closed without merging**